### PR TITLE
Fixing Color picker and other Undefined  errors

### LINKED ISSUE #515: Color Picker not defined in popup.html
**I'm submitting a ...**  
- [x] bug report

**Current behavior:**

As color-picker tag is present only in options.html , thus when I open popup.html, them color picker throws undefined error . 
**Expected behavior:**

Using try catch to find if color picker is present of not.

**Steps to reproduce:**

**Related code:**

```
insert any relevant code here else remove this section
```

**Other information:**

**System information:**

**Screenshots of the issue:**

```
Your operating system
```

**Would you like to work on the issue?**

```
Yes.

![screenshot from 2018-12-28 02-17-15](https://user-images.githubusercontent.com/17084322/50493994-bf996380-0a46-11e9-9659-0783acbbf810.png)

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #515

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

 Fixing Color picker error and other undefined errors

screenshot from 2018-12-28 02-26-36

### CODE REVIEW

**Comment by sk9331657** [comment]
@hkedia321  Please review.

**Comment by sk9331657** [comment]
@zamhaq @hkedia321  @mariobehling Please review.

**Comment by sk9331657** [comment]
@gabru-md @DNS-404 Please review.

**Comment by sk9331657** [comment]
@samagragupta Done. You can check now.

**Comment by sk9331657** [comment]
@Akshat-Jain  @akshatnitd  Please review.

File: src/js/options.js [hunk 1]
```diff
@@ -18,8 +18,14 @@ var restore = document.getElementById("restore");
 var voice;
 var localimage=document.getElementById("localupload");
 let themeColorPickerButton = document.getElementById("btn-color-theme");
-let themeColorPicker = new LdColorPicker(themeColorPickerButton);
-LdColorPicker.init();
+if(themeColorPickerButton) {
+    let themeColorPicker = new LdColorPicker(themeColorPickerButton);
+    LdColorPicker.init();
+    themeColorPicker.on("change", function(color){
```
Comment (fragm3) [inline src/js/options.js]: Use arrow function

File: src/js/options.js [hunk 2]
```diff
@@ -106,22 +112,21 @@ if(backgroundChange) {
...
+if(restore) {
+    restore.addEventListener("click", ()=>{
+        if(confirm("This will remove your customized settings. This action is irreversible.")){
+            userItems = ["theValue", "voice", "msgTheme", "theme"];
+            userItems.forEach(item => {
+                localStorage.removeItem(item);
+            });
+        }
+    });
+
```
Comment (fragm3) [inline src/js/options.js]: Remove new line

**Comment by sk9331657** [comment]
@fragm3  Resolved issue.

**Comment by sk9331657** [comment]
@fragm3  Please review.

**Pull request merged**